config.json

```
@@ -154,6 +154,13 @@
       "method": "cronMethod",
       "cron_frequency": "21600",
       "cron_max_run_time": "3600"
+    },
+    {
+      "cron_name": "cron_data_log",
+      "cron_description": "Cron that generates a downlodable data log file",
+      "method": "cronMethod",
+      "cron_frequency": "1",
+      "cron_max_run_time": "1"
```

```suggestion
      "cron_frequency": "60",
      "cron_max_run_time": "60"
```

Can we set the frequency and max run time to "60" for consistency?

---

I actually just fixed this before seeing your message!
I forgot to put the correct time when submitting it.